### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by sikmir** [tumic0/GPXSee#63]
### Steps to reproduce
```bash
$ lrelease-qt5 gpxsee.pro
$ qmake-qt5 gpxsee.pro
$ make
$ gdb ./GPXSee
(gdb) run
...
Program received signal SIGSEGV, Segmentation fault.
0x000000000047ada9 in Datum::Datum(Ellipsoid const*, double, double, double) ()
(gdb) bt full
#0  0x000000000047ada9 in Datum::Datum(Ellipsoid const*, double, double, double) ()
#1  0x0000000000422c40 in _GLOBAL__sub_I_gcs.cpp ()
#2  0x00000000004afd06 in __do_global_ctors_aux ()
```

### Configuration
GPXSee Version: 5.0
Operating System: Slackware Linux 14.2 (x86_64)
Qt: 5.7.1
GCC: 5.4.0

**Comment by tumic0** [comment]
Was it fixed in 5d590b7c86f9401b5d2f928e723873de1ab6b486? I can't reproduce the crash, but most probably this is it...

**Comment by sikmir** [comment]
@tumic0 yes, it works now.
